appveyor.yml

```diff
@@ -59,6 +59,10 @@ init:
 clone_depth: 1
 
 install:
+ # Activate the NvDA build system Python virtual environment, creating / updating it if necessary.
+ - .\venvUtils\ensureAndActivate.bat
+ # Save an exact list of the actual Python packages and their versions that got installed, to aide in reproducing a build
+ - py -m pip freeze >output\pip_packages.txt

```

Comment:
At this point there is no directory called 'output' - it is created later on by SCons, so this command fails.